lib/worker/src/db/publications.rs

```diff
@@ -103,15 +105,24 @@ pub async fn delete_expired_job(pool: &PgPool) -> Result<()> {
 // Lists cids of a given publication
 pub async fn pub_cids(
```

Comment:
refactors this method to accept `Vault` as input, and to return `RecordInfo` instead of a tuple

Reply:
do we want to change the file name to `vault.rs`?